riscv/insns/amoadd_b.h

```diff
@@ -0,0 +1,3 @@
+require_extension('A');
```

Comment:
We don't need this, because you already imposed the Zabha -> A dependence in `isa_parser_t` above.

Reply:
Removed